init_scripts/Dockerfile_VEP105

```diff
@@ -0,0 +1,16 @@
+FROM ensemblorg/ensembl-vep:release_105.0
+
+USER root
+
+RUN apt-get update && apt-get -y install   wget   libncurses5-dev   libncursesw5-dev   libbz2-dev   liblzma-dev   sqlite3   libsqlite3-dev   cpanminus   git &&   rm -rf /var/lib/apt/lists/*
+
+RUN cpanm DBD::SQLite
+
+RUN wget https://github.com/samtools/samtools/releases/download/1.7/samtools-1.7.tar.bz2 &&   tar xjvf samtools-1.7.tar.bz2 &&   cd samtools-1.7 &&   make && make install
+
+WORKDIR /opt/vep/
+
+RUN git clone https://github.com/Ensembl/VEP_plugins.git Plugins/
+RUN mkdir /opt/vep/Plugins/loftee && git clone -b grch38 https://github.com/konradjk/loftee.git && mv /opt/vep/loftee/* /opt/vep/Plugins/loftee
```

Comment:
Just confirming this works since it is different from the previous docker file, and the init file has `loftee_path:/opt/vep/Plugins/`